pip/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+authors = ["Microsoft"]
+name = "pyqir_runner"
```

Comment:
I always forget which one's preferred, but we use `-` for our other packages (`azure-quantum`, `qsharp-widgets` etc)

Also, unless this name has already been established, I think `qir-runner` is a nicer name, the `py` is implied by this being a Python package ;)